### ISSUE TITLE
Documentation refresh & optional training

### ISSUE THREAD

**Issue by leRoe93** [VISAB-ORG/VISAB#31]
I would like to add some enhanced documentation to the project. For example package-info for each package to denote its actual purpose at least.

Furthermore I will draw a graph that tries to cover all relevant stuff from the VISAB project so all team members have an easier entry point for any kind of task (I also need to have a deeper look into some stuff from @VocalTrance haha).

@TimKuno @Marshmellow24 @vanessas97 

Please discuss within this issue, which kind of things across VISAB may seem unclear and we should get rid of some barriers. 

If somebody wants it, we could also again do some kind of refreshing training on VISAB / VisabConnector (C#) / VisabConnector (Java) to cover stuff which appeared additonally during the last weeks. 

To sum up: The outcome will be an enhanced documentation within the VISAB project as well as some UML-style graph pointing out relevant structures and workflows.
